website/content/en/status/report-2025-01-2025-03/clusteradm.adoc

```diff
@@ -0,0 +1,61 @@
+
+In this quarter, the team has worked on the following:
+
+* Regular support for FreeBSD.org user accounts.
+* Regular disk and parts support (and replacement) for all physical hosts and mirrors.
+* Cluster software refresh.
+* Moving more cluster services to Chicago.
+* Supporting the Grimoirelab dashboard effort.
+* Coordinate community mirrors.
+
+==== Moving cluster services to Chicago
+
+We started building up our new site in Chicago 2024, with a long-term goal to have Chicago as our primary location.
+Since 2024Q4, we began decommissioning older machines in New Jersey and moving services to the newer machines in Chicago.
+In 2025Q1, we started upgrading critical services in the cluster and testing to setup in Chicago.
+
+==== Git web interface mirrors
+
+While the project's public read-only git repository is built by a globally distributed mirror, the web interface (cgit) is not.
+We found there is increasing requirement of accessing it, and for improving the response time and reilibility, we setup the cgit on the mirrors around the world.
+
+==== FreeBSD official mirrors
+
+Current locations are Australia, Brazil, Germany, Japan (two full mirror sites), Malaysia, South Africa, Sweden, Taiwan, United Kingdom (full mirror site), United States of America -- California, Chicago, New Jersey (primary site), and Washington.
+
+Our mirror site in Taiwan is experiencing an extended outage.
+The effort of bringing it back is in progress.
+We hope to have it back online during the second quarter of 2025.
+
+The hardware and network connection have been generously provided by:
+
+* Cloud and SDN Laboratory at link:https://www.bbtower.co.jp/en/corporate/[BroadBand Tower, Inc]
+* link:https://www.cs.nycu.edu.tw/[Department of Computer Science, National Yang Ming Chiao Tung University]
+* link:https://deploy.equinix.com/[Equinix]
+* link:https://internet.asn.au/[Internet Association of Australia]
+* link:https://www.isc.org/[Internet Systems Consortium]
+* link:https://www.inx.net.za/[INX-ZA]
+* link:https://www.kddi-webcommunications.co.jp/english/[KDDI Web Communications Inc]
+* link:https://www.mohe.gov.my/en/services/research/myren[Malaysian Research & Education Network]
+* link:https://www.metapeer.com/[MetaPeer]
+* link:https://www.nyi.net/[New York Internet]
+* link:https://nic.br/[NIC.br]
+* link:https://www.teleservice.net/[Teleservice Skåne AB]
+* link:https://your.org/[Your.Org]
+
+New official mirrors are always welcome.
+We have noted the benefits of hosting single mirrors at Internet Exchange Points globally, as evidenced by our existing mirrors in Australia, Brazil, and South Africa.
+If you are affiliated with or know of any organizations willing to sponsor a single mirror server, please contact us.
+We are particularly interested in locations on the United States West Coast and throughout Europe.
```

Comment:
Please add * link:https://sonic.net[Sonic]

We're good for mirrors on the west coast of America.  Definitely still need capacity in Europe though.

(Yes, I still need to finish setting up the Sonic mirror.  I also need to work on Fastly.  Not enough progress to mention there.)